test/routes/misc/templates.test.ts

```diff
@@ -0,0 +1,14 @@
+import { renderTemplate } from '@/templates';
+
+describe('templates', () => {
+  it('should render proper sms template', async () => {
+    const template = 'signin-passwordless-sms';
+    const message = await renderTemplate(`${template}/text`, {
+      locale: 'en',
+      displayName: 'John Doe',
+      code: '123456',
+    });
+
+    expect(message).toEqual('Your code is 123456.');
+  });
```

Comment:
Please add a test for `locale: 'fr'` too to ensure we're not always using the fall-back message if `message` is empty.

Reply:
Done! Looking forward! Thanks!